packages/actor-query-source-identify-hypermedia-annotate-source/test/ActorQuerySourceIdentifyHypermediaAnnotateSource-test.ts

```diff
@@ -87,8 +87,8 @@ describe('ActorQuerySourceIdentifyHypermediaSourceAttribution', () => {
       ]);
       context = context.set(KEY_CONTEXT_WRAPPED, true);
       await expect(actor.test({ metadata: <any> null, quads, url: 'URL', context }))
-        .rejects
-        .toThrow('Unable to wrap query source multiple times');
+        .resolves
+        .toEqual(failTest('Unable to wrap query source multiple times'));
```

Comment:
We have a `toFailTest` helper.
You can see an example here: https://github.com/comunica/comunica/blob/6104e3b5c3c1171684e9e12e1443f717ca46619a/packages/actor-http-proxy/test/ActorHttpProxy-test.ts#L57

Reply:
Right, fixed.